source/kubernetes/quickstart.rst

```diff
@@ -132,34 +126,19 @@ can also be configured individually. For this guide we will use the default valu
 
 Select the **Network** tab to configure the cluster's network access.
 
-A few options are available here, but for this guide, just make sure that
-**Create New Network** is checked (the default), to create a new virtual network for the cluster.
+To enable access to the Kubernetes API from the public Internet, select
+'Accessible with public floating IP' from the **Floating IP** drop-down list.
+For the quickstart, we will leave the **Allowed CIDRs** field empty.
 
 .. image:: _containers_assets/quickstart-network.png
     :align: center
 
-Finally, select the **Advanced** tab to set one more option.
-
-In order to be able to access our cluster from outside the
-virtual network we will need to make sure that the Kubernetes
-API is available from the public Internet. To do this,
-add the following label to **Additional Labels**:
-
-.. code-block:: bash
-
-  master_lb_floating_ip_enabled=true
-
-The form should be filled out like this:
-
-.. image:: _containers_assets/quickstart-advanced.png
-    :align: center
-
-That should be everything you need to configure,
+That is everything you need to configure for this guide,
 so press the **Submit** button to create the cluster.
 
 You will be returned to the **Clusters** page, where you can monitor the state
 of your Kubernetes clusters. Our new cluster should be listed,
-in ``CREATE_IN_PROGRESS`` state.
+in ``CREATE_IN_PROGRESS`` state, refresh the browser to see the updated status.
```

Comment:
```suggestion
in ``CREATE_IN_PROGRESS`` state. If the cluster is not yet listed, refresh the page.
```

Reply:
Even if it is listed, the status won't update to `CREATE_COMPLETE` without refreshing. I'll have a think about the wording on this.

Reply:
You're right. Guess we should also consider putting in an auto-refresh for this on the dashboard side, as well (if possible).

Reply:
Updated